.env.example

```diff
@@ -0,0 +1,46 @@
+APP_NAME=Escooters
+APP_ENV=local
+APP_KEY=
```

Comment:
I would put generated app key in exmaple envfile.

Reply:
added